src/EnergyPlus/CMakeLists.txt

```diff
@@ -614,7 +613,6 @@ set(SRC
     ThermalEN673Calc.hh
     ThermalISO15099Calc.cc
     ThermalISO15099Calc.hh
-    Timer.h
```

Comment:
Removed the internal timing stuff per some discussion here: https://github.com/NREL/EnergyPlus/discussions/9468.  Note that this does not affect the user-facing runtime tracking that shows up in the output files.  This is merely removing the compiler-define-dependent, manual, internal time tracking, that is not necessary given modern profiling tools.